bots/pr/src/main/java/org/openjdk/skara/bots/pr/IssueCommand.java

```diff
@@ -98,6 +107,170 @@ private void showHelp(PrintWriter reply) {
+                reply.println("Updating description of additional solved issue: `" + issue.toShortString() + "`.");
+            } else {
+                reply.println("Adding additional issue to " + name + " list: `" + issue.toShortString() + "`.");
+            }
+        }
+    }
+
+    private void removeIssue(PullRequestBot bot, String args, Set<String> currentSolved, PrintWriter reply) throws InvalidIssue {
+        var issueListStart = args.indexOf(' ');
+        if (issueListStart == -1) {
+            showHelp(reply);
+            return;
+        }
+        if (currentSolved.isEmpty()) {
+            reply.println("This PR does not contain any additional solved issues that can be removed. To remove the primary solved issue, simply edit the title of this PR.");
+            return;
+        }
+        var issuesToRemove = parseIssueList(bot.issueProject() == null ? "" : bot.issueProject().name(), args.substring(issueListStart));
+        for (var issue : issuesToRemove) {
+            if (currentSolved.contains(issue.shortId())) {
+                reply.println(SolvesTracker.removeSolvesMarker(issue));
+                reply.println("Removing additional issue from " + name + " list: `" + issue.shortId() + "`.");
+            } else {
+                reply.print("The issue `" + issue.shortId() + "` was not found in the list of additional solved issues. The list currently contains these issues: ");
+                var currentList = currentSolved.stream()
+                                               .map(id -> "`" + id + "`")
+                                               .collect(Collectors.joining(", "));
+                reply.println(currentList);
+            }
+        }
+    }
+
+    private void createIssue(PullRequestBot bot, PullRequest pr, String args, CensusInstance censusInstance, HostUser author, PrintWriter reply) {
+        if (!censusInstance.isAuthor(author)) {
+            reply.println("Only [Authors](https://openjdk.java.net/bylaws#author) are allowed to create issues.");
+            return;
+        }
+
+        var currentTitleIssue = Issue.fromString(pr.title());
+        if (currentTitleIssue.isPresent()) {
+            reply.println("The PR title already references an issue (`" + currentTitleIssue.get().shortId() + "`) - will not create a new one.");
+            return;
+        }
+
+        var argSplit = new LinkedList<>(Arrays.asList(args.split("(?:\\s+|/)")));
+        argSplit.pollFirst();
+
+        String priority = null;
+        String subComponent = null;
+
+        // First argument can be a priority
+        var next = argSplit.pollFirst();
+        if (next != null && next.matches("^[pP]\\d$")) {
+            priority = next.substring(1);
+            next = argSplit.pollFirst();
+        }
+
+        // Next argument is the mandatory component name
+        if (next == null) {
+            showHelp(reply);
+            return;
+        }
+        var component = next;
```

Comment:
Given that you incorporated my changes to the help text I think you need to add some code here to handle the case when `component` is something like `"hotspot/gc"` :smiley:

Reply:
But it does work, there's even a test for it. :)

Reply:
Sigh, I didn't notice the regular expression in `args.split("(?:\\s+|/)"))` :sleeping: Looks good then :+1: